### ISSUE TITLE
Update to 0.4.3 is failing

### ISSUE THREAD

**Issue by M1ch43lV** [lightningequipment/circuitbreaker#67]
After updating circuitbreaker to 0.4.3, I get the message 
".gitkeep" when accessing cb through the web ui.
Is there some reset I can do?

**Comment by joostjager** [comment]
How exactly did you update? Are you running the docker image?

If you build from source, you may need to build the frontend by running `./build_frontend.sh`

**Comment by M1ch43lV** [comment]
I am using a non-docker environment (raspi-/minibolt) and did an update of the master branch.
Unfortunately the build_frontend is for a docker environment, can you provide one for non-docker environment?

**Comment by jjager-nydig** [comment]
It is difficult because of all the build dependencies required that then would need to be installed locally (nodejs, yarn). In https://github.com/lightningequipment/circuitbreaker/blob/master/Dockerfile you can see the command to run for building the frontend.

Alternatively you can try building on a different machine that does have docker, and then copy over the frontend files.

**Comment by M1ch43lV** [comment]
I see. Can you give me a hint to which commit version I can revert back without the need to install docker? 
   The last version I used was Commits on Feb 13, 2023  1568c16482cedf366183110025d4ce53b95de79b
   
   
   
    jjager-nydig ***@***.***> hat am 04.04.2023 08:25 CEST geschrieben:
    
   
     
    
   
     
    
     
   It is difficult because of all the build dependencies required that then would need to be installed locally (nodejs, yarn). In https://github.com/lightningequipment/circuitbreaker/blob/master/Dockerfile you can see the command to run for building the frontend. 
   Alternatively you can try building on a different machine that does have docker, and then copy over the frontend files.

**Comment by M1ch43lV** [comment]
For all the people with similar problem using a non-docker linux version. This is the last commit 17a443b9d5b661c62cc2f3362d8d21030da7bda3 working with the given web-build.
Manual steps to update: 
```
cd circuitbreaker
git fetch
git checkout 17a443b
go install
```
However this should be considered as temporary solution.

**Comment by jjager-nydig** [comment]
An intermediate solution would be to go back to checking in the frontend build, but add a CI step that verifies that the checked-in build is up to date.

**Comment by joostjager** [comment]
Fixed by #73
